### DISCUSSION TITLE
Not able to load mod_authz_oidc

### DISCUSSION THREAD

**Discussion by Amrutapan** [OpenIDC/mod_auth_openidc#511] [Q&A] [answered]
Hi ,
I got below error while loading oidc module:
Syntax error on line 1033 of /home/persistent.co.in/unity_svn/Middleware/Oracle_WT1/instances/instance1/config/OHS/ohs1/httpd.conf: Syntax error on line 1 of /home/persistent.co.in/unity_svn/Middleware/Oracle_WT1/instances/instance1/config/OHS/ohs1/moduleconf/auth_openidc.conf: Cannot load /home/persistent.co.in/unity_svn/Middleware/Oracle_WT1/ohs/modules/mod_auth_openidc.so into server: /home/persistent.co.in/unity_svn/Middleware/Oracle_WT1/ohs/modules/mod_auth_openidc.so: **undefined symbol: ap_get_exec_line**
Please help me to find a cause.
I have used mod_auth_openidc-2.4.5-1.el7.x86_64rpm.
linux 7
OHS 2.4

**Comment by zandbelt** [comment, marked as answer]
Oracle HTTP server is only supported under a commercial subscription
